### PR TITLE
About IoPay menu item not showing info dialog #1434

### PR DESCRIPTION
**Pull request by liaojikunwork**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```
Screen Shot 2020-08-04 at 1 38 01 PM

### CODE REVIEW

**Comment by liaojikunwork** [comment]
Screen Shot 2020-08-04 at 2 00 53 PM

You can see it on the Windows Instance.

File: src/electron/src/about.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<body>
+  <img alt="logo" class="logo" src="./icon.png" />
+  <p class="text-center title">ioPay</p>
+  <p class="text-center version">Version 0.10.0 (0.10.0)</p>
```
Comment (guo) [inline src/electron/src/about.html:4]: change the version to dynamic from the package.

the copyright too.

Comment (liaojikunwork) [reply src/electron/src/about.html:4]: @guo  I can get version info dynamically from the package, but can't for the copyright field.

File: src/electron/package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     }
   },
   "author": "Tian Pan (https://tianpan.co)",
-  "license": "Apache",
+  "license": "Copyright © 2020 IoTeX Foundation",
```
Comment (guo) [inline src/electron/package.json:50]: so, this is the license. you should change add a copyright line or reuse one. License should be the same.

Comment (guo) [inline src/electron/package.json:50]: you can use author if not found copyright.

Comment (liaojikunwork) [reply src/electron/package.json:50]: Ok

Comment (liaojikunwork) [reply src/electron/package.json:50]: https://github.com/iotexproject/iotex-explorer/pull/1459/files

**Pull request merged**